### DISCUSSION TITLE
I noticed that loc_loss uses l1 loss instead of smooth l1 loss. Does l1 loss perform better?

### DISCUSSION THREAD

**Discussion by TCBocean** [STVIR/pysot#209] [General]
As the title says, does the l1 loss have a better performance than the smooth l1 loss, is there any relevant data? Because in your paper, the regression loss uses a smooth l1 loss.
I am looking forward to someone who can answer me, thank you.

**Comment by chun-chun123** [comment]
https://www.jianshu.com/p/19483787fa24

**Comment by TCBocean** [comment]
@chun-chun123 
Yes, as the article says, smoothL1 has many advantages. 
https://github.com/STVIR/pysot/blob/052b9678a7ed336752f74dc6af31cc00eb004551/pysot/models/loss.py#L31
But I saw in the code that L1 loss was used, so I asked this question.
